Bio/Align/Applications/_ClustalOmega.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Nice that the tool catches there - Python 3 defaults to UTF8, which is great for using unicode directly in string literals.

Reply:
I pull out the five cases of this as #2457 to help break down this monster pull request into more manageable chunks.